### PR TITLE
Bugfixes / changes

### PR DESCRIPTION
**Pull request by sfk-steelsong**
* Fixed issues with sending DMs for admin commands (I changed some of them to act like they used to before the change)
* cleaning up rules logic to make it more obvious what's happening and give more time to read
* updated checklist text for boxes and crates to clarify how to get them, and changed crates to be from any battle

### CODE REVIEW

File: src/utils/sender.js [hunk 1]
```diff
@@ -136,7 +136,12 @@ exports.msgUser = async function(id,msg){
 		return;
 	}
 	let user = await channel.recipient;
-	if(channel) await channel.createMessage(msg);
+	try{
+		if(channel) await channel.createMessage(msg);
+	}catch(err){
+		user.dmError = true;
```
Comment (ChristopherBThai) [inline src/utils/sender.js:142]: We're altering the Eris user object. The user will keep `dmError` as true until the bot resets.

**Review by ChristopherBThai — CHANGES_REQUESTED** [review]

File: src/utils/sender.js (hunk 1 above)
Comment (sfk-steelsong) [reply src/utils/sender.js:142]: oh fair, I can bubble up the try/catch logic.  although setting it to `true` after the createMessage call _might_ be easier... Unless you don't want me to throw that on there.

Comment (ChristopherBThai) [reply src/utils/sender.js:142]: I would prefer not to alter the actual object since we're keeping that data for awhile.

Comment (sfk-steelsong) [reply src/utils/sender.js:142]: Okay, let me see what I can do. it'll be a more comprehensive rework of code though.

Comment (ChristopherBThai) [reply src/utils/sender.js:142]: What about a shallow copy of the user object? We can use either `Object.assign({ dmError: true }, user)` or `{ dmError: true, ...user }`. I need to look at the efficiency difference of these two though. Give me a sec

Comment (sfk-steelsong) [reply src/utils/sender.js:142]: basically, just need to get something back to the calling method.  Maybe I can return a falsey value for error / null for anything else (guild, channel, etc)

**Pull request merged**